rules/no-send.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * @fileoverview Discourage use of 'send' as it is unsafe
+ * @author Tristan Homsi <tristanhomsi@gmail.com>
+ */
+
+'use strict';
+
+function isSend (calleeNode) {
+	return (calleeNode.property.type === 'Identifier' &&
+		calleeNode.property.name === 'send'
+	);
+}
+
+module.exports = {
+
+	meta: {
+
+		docs: {
+			recommended: true,
+			type: 'warning',
+			description: 'Ensure no use of \'send\' in the code'
+		},
+
+		schema: [],
+
+		fixable: 'code'
```

Comment:
remove `fixable: 'code'` - this property is set **only** if the rule supplies at least 1 fix.

Reply:
thanks, good catch :) I had a fixable in before (changing `send` to `transfer`) but I removed it

Reply:
I thought that must've been the case (has happened with me too!)